config/local.json

```diff
@@ -0,0 +1,12 @@
+{ 
+  "capabilities": {
```

Comment:
Let's have the caps in W3C format by default on main branch, can keep the legacy caps in a different branch.

Reply:
fixed https://github.com/browserstack/pytest-browserstack/pull/2/commits/4eb3e554a7d2cdf532ee88f11b0d3c09520b27db